store/src/java/com/zimbra/qa/unittest/TestModifyDynamicEphemeralAttrs.java

```diff
@@ -0,0 +1,62 @@
+package com.zimbra.qa.unittest;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.zimbra.cs.account.Account;
+import com.zimbra.cs.account.Provisioning;
+
+public class TestModifyDynamicEphemeralAttrs {
+
+    private static final String acctName = TestModifyDynamicEphemeralAttrs.class.getSimpleName();
+    private static Account acct;
+    private static Provisioning prov = Provisioning.getInstance();
+
+    @BeforeClass
+    public static void init() throws Exception {
+        if (!TestUtil.accountExists(acctName)) {
```

Comment:
I think it is better to delete the account before the test even if it exists already in case there is some garbage in the account.